src/modules/claims/claims.service.ts

```diff
@@ -1354,6 +1404,23 @@ export class ClaimsService {
     );
   }
 
+  /**
+   * Verifies that credential was issued by authorized issuer
+   *
+   * @param {VerifiableCredential<RoleCredentialSubject} vc to be verified
+   * @return void. Returns "Proof Not Verified" error if VC not verified. Returns error if issuer not verified
+   */
+  async verifyVc(
```

Comment:
```suggestion
  async verify(subjectDID, roleName) {
      // resolveCredential implementation in ew-credentials
  // pratically, current approach is to resolve DID doc
  // and use serviceEndpoints to pull credentials from IPFS
  const credential = resolveCredential(subjectDID, roleName);
  if (credential is VC) {
    verifyVc(vc);
  }
  else if (credential is offChainClaim) {
    verifyOffChainClaim
   }
  }
 ```

Reply:
@jrhender @JGiter does this look like what we agreed upon for revising this? one entry point with subject DID and role. Do we have code in place for verifyOffChainClaim? I don't think we do, so that would be part of this PR?

Reply:
Should it be subjectDID (the subject of the credential) that we are passing in, or should it be the issuer? It seems this method is to validate the issuer?

Reply:
@whitneypurdum I think we should keep method signature as is